aten/src/ATen/native/native_functions.yaml

```diff
@@ -9961,6 +9961,19 @@
   dispatch:
     DefaultBackend: det
 
+- func: linalg_slogdet(Tensor self) -> (Tensor sign, Tensor logabsdet)
+  python_module: linalg
+  use_c10_dispatcher: full
+  variants: function
+  dispatch:
+    DefaultBackend: linalg_slogdet
```

Comment:
We should probably just use CPU, CUDA instead of DefaultBackend here?

Reply:
Yep, we can start using `CPU, CUDA`. Many other linalg function use DefaultBackend now and I followed the pattern here.